changelog/splitting_format.dd

```diff
@@ -0,0 +1,15 @@
+Module `std.format` splitted into smaller modules
+
+The module `std.format` has been splitted into smaller modules:
+
+$(UL
+    $(LI `std.format.spec`: Stuff concerning the format string, mainly
+         the `struct FormatSpec` and the `template singleSpec`)
+    $(LI `std.format.internals`: Several modules containing non-public
+         stuff)
```

Comment:
`submodules containing private symbols`, although I'm not sure theres much need to draw attention to internal submodules.

Reply:
I added them for completeness, but without any details.